core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol

```diff
@@ -52,21 +51,23 @@ contract ExpiringMultiPartyCreator is ContractCreator, Testable, Lockable {
     AddressWhitelist public collateralTokenWhitelist;
     // - Address of TokenFactory to pass into newly constructed ExpiringMultiParty contracts
     address public tokenFactoryAddress;
+    // - Address of contract that keeps track of time for testing environments, or 0x0 for production that use live time.
+    address public timerContractAddress;
```

Comment:
nit: aren't we already storing this in the Testable base class? Can't we just grab it from there?

Reply:
yup you're totally right